subsquid/src/model/generated/pabloPool.model.ts

```diff
@@ -47,4 +47,16 @@ export class PabloPool {
 
     @OneToMany_(() => PabloAssetWeight, e => e.pool)
     poolAssetWeights!: PabloAssetWeight[]
+
+    /**
+     * Last updated block id
+     */
+    @Column_("text", {nullable: false})
+    blockId!: string
+
+    /**
+     * Asset to be used as reference when calculating the pool's TVL
+     */
+    @Column_("text", {nullable: false})
+    baseAssetId!: string
```

Comment:
```suggestion
    quoteAssetId!: string
```

Reply:
This represents the asset to be used when normalizing values for a given pool. Naming it `quote` would be more accurate?

Reply:
yes, `quote` is the correct naming for an asset when it is used for quoting amounts